README.md

```diff
@@ -20,4 +20,19 @@
     `node server.js`
 
 - Hit [http://localhost:3000](http://localhost:3000) on your favorite browser.
-- Learn and hack it!
\ No newline at end of file
+- Learn and hack it!
+
+## Usage With Docker
+- Make sure you have installed docker and docker-compose on your machine
```

Comment:
Give highlight background to `docker` & `docker-compose` words. Wrap the words with back tick.